Upstream commit f8ed2998bffe481ab267d20dc2b641acbffbf021

Remove the Apple iOS PNG variant

This added a stat and magic check for each and every PNG file around.

This is particularly a problem for the icon cache, as each PNG file
in a theme would end up being checked for which one of the variants
it was.

diff --git a/freedesktop.org.xml.in b/freedesktop.org.xml.in
index 55d550f..cb5fd0c 100644
--- a/freedesktop.org.xml.in
+++ b/freedesktop.org.xml.in
@@ -4303,15 +4303,6 @@ command to generate the output files.
     </magic>
     <glob pattern="*.png"/>
   </mime-type>
-  <mime-type type="image/x-apple-ios-png">
-    <_comment>Apple optimised PNG image</_comment>
-    <magic priority="80">
-      <match type="string" value="\x89PNG" offset="0">
-        <match value="CgBI" type="string" offset="12"/>
-      </match>
-    </magic>
-    <glob weight="10" pattern="*.png"/>
-  </mime-type>
   <mime-type type="image/rle">
     <_comment>Run Length Encoded bitmap image</_comment>
     <glob pattern="*.rle"/>
-- 
cgit v0.10.2

